client/my-sites/customer-home/cards/tasks/connect-accounts.jsx

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import React, { useState } from 'react';
+import { useTranslate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { Button } from '@automattic/components';
+import { isMobile } from '@automattic/viewport';
+
+/**
+ * Internal dependencies
+ */
+import ActionPanel from 'components/action-panel';
+import ActionPanelTitle from 'components/action-panel/title';
+import ActionPanelBody from 'components/action-panel/body';
+import ActionPanelFigure from 'components/action-panel/figure';
+import ActionPanelCta from 'components/action-panel/cta';
+import Gridicon from 'components/gridicon';
+import QueryPublicizeConnections from 'components/data/query-publicize-connections';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import { savePreference } from 'state/preferences/actions';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const ConnectAccountsTask = ( { skipTask, siteSlug } ) => {
+	const translate = useTranslate();
+	const [ isVisible, setIsVisible ] = useState( true );
+
+	if ( ! isVisible ) {
+		return null;
+	}
+	return (
+		<ActionPanel className="tasks connect-accounts">
+			<QueryPublicizeConnections selectedSite />
+			<ActionPanelBody>
+				{ ! isMobile() && (
+					<ActionPanelFigure align="right">
+						<img src="/calypso/images/stats/tasks/social-links.svg" alt="" />
+					</ActionPanelFigure>
+				) }
+				<div className="tasks__timing">
+					<Gridicon icon="time" size={ 18 } />
+					<p>{ translate( '3 minutes' ) }</p>
+				</div>
+				<ActionPanelTitle>{ translate( 'Drive traffic to your site' ) }</ActionPanelTitle>
```

Comment:
Out of curiosity, is there any reason why you used "Drive traffic to your site" instead of "Connect social media accounts" as shown on #40734? Both read good to me though.

Reply:
Yep, that was the language used in the design posted here: https://github.com/Automattic/wp-calypso/pull/41018#discussion_r406949061